packages/trpc/server/routers/viewer/organizations/adminGet.handler.ts

```diff
@@ -0,0 +1,60 @@
+import { prisma } from "@calcom/prisma";
+import { teamMetadataSchema } from "@calcom/prisma/zod-utils";
+
+import { TRPCError } from "@trpc/server";
+
+import type { TrpcSessionUser } from "../../../trpc";
+import type { TAdminGet } from "./adminGet.schema";
+
+type AdminGetOptions = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: TAdminGet;
+};
+
+export const adminGetHandler = async ({ input }: AdminGetOptions) => {
+  const org = await prisma.team.findUnique({
+    where: {
+      id: input.id,
+      AND: [
```

Comment:
Just a little idea i had when reading this - i wonder if it would be easier to query by  just team ID -> parse metadata -> check its set and throw from there. 

(Probably ms saved no biggy)

Reply:
Yeah, makes a lot of sense actually.

Reply:
Fixed